### PR TITLE
Upgrade to 7.1.0

### PR DESCRIPTION
**Pull request by henzigo**
| Q             | A
| ------------- | ---
|Description, reason for the PR| Upgrade demoshop to shopsys framework 7.1.0
|New feature| Yes 
|BC breaks| Yes 
|Fixes issues| ...
|Standards and tests pass| Yes
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: build.xml [hunk 1]
```diff
@@ -52,10 +52,10 @@
...
-    <target name="build-deploy-part-2-db-dependent" depends="db-migrations,create-domains-data,generate-friendly-urls,replace-domains-urls,grunt,error-pages-generate,warmup" description="Second part of application build for production preserving your DB (must be run with maintenance page when containing DB migrations)."/>
-    <target name="build-new" depends="wipe,redis-check,clean-redis,composer,npm,dirs-create,assets,db-rebuild,grunt,error-pages-generate,warmup,product-search-create-structure" description="Builds application for production with clean DB (with base data only)."/>
-    <target name="build-demo" depends="wipe,redis-check,clean-redis,composer,npm,dirs-create,assets,db-demo,grunt,error-pages-generate,warmup,product-search-recreate-structure,product-search-export-products" description="Builds application for production with clean demo DB."/>
+    <target name="build-deploy-part-1-db-independent" depends="clean,redis-check,composer,npm,dirs-create,assets" description="First part of application build for production preserving your DB (can be run without maintenance page)."/>
```
Comment (boris-brtan) [inline build.xml]: please check PR for this file https://github.com/shopsys/shopsys/pull/886/files#diff-e22ff16d006c03464bffffa8462c123a
there is generate-build-versions for `build-deploy-part-1-db-independent`, `build-new`, `build-demo`
it is this instruction
``- and include generate-build-version and clean-redis-old to your build phing targets. Please find inspiration in [#886]``
if needed make update into `upgrade-v7.1.0.md`

**Review by boris-brtan — CHANGES_REQUESTED** [review]
Hi,
THX for the PR i just see one thing that seems to be missing.
Maybe some rephrasement of changelog things could be better for consistency but i dont insist on it.

File: build-dev.xml [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+    <target name="build-demo-dev" depends="generate-build-version,wipe-excluding-logs,redis-check,clean-redis-old,composer-dev,npm,timezones-check,dirs-create,test-dirs-create,assets,db-demo,product-search-recreate-structure,product-search-export-products,grunt,error-pages-generate,tests-acceptance-build,checks-diff" description="Builds application for development with clean demo DB and runs checks on changed files."/>
+    <target name="build-demo-dev" depends="generate-build-version,wipe-excluding-logs,redis-check,clean-redis-old,composer-dev,npm,timezones-check,dirs-create,test-dirs-create,assets,db-demo,product-search-recreate-structure,product-search-export-products,grunt,error-pages-generate,tests-acceptance-build,checks-diff" description="Builds application for development with clean demo DB and runs checks on changed files."/>
```
Comment (boris-brtan) [inline build-dev.xml]: hmm it looks like build-demo-dev is duplicated since unknown date
```suggestion
```

File: build.xml (hunk 1 above)
Comment (henzigo) [reply build.xml]: I forgot to update that targets. I updated it in fixup

File: build-dev.xml (hunk 2 above)
Comment (henzigo) [reply build-dev.xml]: I removed that target in new commit

**Review by boris-brtan — APPROVED** [review]

**Pull request merged**